internal/view/content.html

```diff
@@ -48,15 +50,26 @@
 			el: '#content-scene',
 			mixins: [basePage],
 			data: {
-				modified: "$$.Book.Modified$$"
+				created: "$$.Book.Created$$"
 			},
 			methods: {
-				localtime() {
-					var strTime = this.modified.replace(" ", "T");
-					if (!strTime.endsWith("Z")) {
-						strTime += "Z";
+				createdtime() {
```

Comment:
Can we use `camelCase` here?
```suggestion
				createdTime() {
```